docs/tutorials/timeseries/forecasting-indepth.md

```diff
@@ -0,0 +1,248 @@
+
+splitter = MultiWindowSplitter(num_windows=5)
+predictor = TimeSeriesPredictor(..., validation_splitter=splitter)
+```
+
+Alternatively, a user can provide their own validation set to the `fit` method and forego using the splitter completely. In this case it's important to remember that the validation score is computed on the last `prediction_length` time steps of each time series.
+```
+predictor = TimeSeriesPredictor(...)
+predictor.fit(train_data=train_data, tuning_data=my_validation_dataset)
+```
+
+## What functionality does `TimeSeriesPredictor` offer?
+AutoGluon offers multiple ways to configure the behavior of a `TimeSeriesPredictor` that are suitable for both beginners and expert users.
+
+### Basic configuration with `presets` and `time_limit`
+We can fit `TimeSeriesPredictor` with different pre-defined configurations using the `presets` argument of the `fit` method.
+
+```python
+predictor = TimeSeriesPredictor()
+predictor.fit(train_data=train_data, presets="medium_quality")
+```
+
+Higher quality presets, in general, result in better forecasts but take longer to train.
+The following presets are available:
+
+**TODO: These will be significantly changed by 0.6.0**
+
+- `"low_quality"`: quickly train a few toy models. This setting should only be used as a sanity check.
+- `"medium_quality"`: train several selected models (`"ETS"`, `"ARIMA"`, `"DeepAR"`, `"SimpleFeedForward"`) without hyperparameter optimization. A good baseline setting.
+- `"high_quality"`: same as `"medium_quality"`, but with an extended model zoo (+ `"MQRNN"`, `"Transformer"`, `"TemporalFusionTransformer"`).
+- `"best_quality"`: Train all available models with hyperparameter optimization.
+
+Another way to control the training time is using the `time_limit` argument.
+
+```python
+predictor.fit(
+    train_data=train_data,
+    time_limit=60 * 60,  # total training time in seconds
+)
+```
+
+If no `time_limit` is provided, the predictor will train until all models have been fit.
+
+
+### Manually specifying what models should be trained
+Advanced users can override the presets and manually specify what models should be trained by the predictor using the `hyperparameters` argument.
+
+
+```python
+predictor = TimeSeriesPredictor()
+
+predictor.fit(
+    train_data=train_data,
+    hyperparameters={
+        "DeepAR": {},
+        "ETS": {
+            "seasonality": "add",
+            "seasonal_period": 7,
+        }
+    }
+)
+```
+The code above will only train two models:
+
+- `DeepAR` (with default hyperparameters)
+- `ETS` (with the given `seasonality` and `seasonal_period`; all other parameters set to their defaults).
+
+For the full list of available models and the respective hyperparameters, see [Model zoo](#TODO).
+
+### Hyperparameter tuning
+Advanced users can define search spaces for model hyperparameters and let AutoGluon automatically determine the best configuration for the model.
+
+```python
+import autogluon.core as ag
```

Comment:
I'd prefer to avoid this notation, instead import the Spaces namespace.

Reply:
Do you mean the following notation (as in Tabular - In-depth tutorial)?
```python
import autogluon.core as ag

ag.space.Categorical(1, 2, 3)
```
or this one?
```python
from autogluon.core import space

space.Categorical(1, 2, 3)
```

Reply:
probably as in Tabular - In-depth tutorial